src/french.js

```diff
@@ -52,7 +52,46 @@ class French {
       }
   */
   getInfoList () {
-    return allInfo;
+    return Object.keys(allInfo);
```

Comment:
returning allInfo is better. Component is iterating through object to build options.